### PR TITLE
Add axis in bondwire

### LINKED ISSUE #3214: Bug located in create_bondwire part 2
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

In create_bondwire the "WhichAxis" property should be available as a parameter in order to change the orientation of the wire bondings. By default, "WhichAxis" is declared in the "Z" axis direction. X, Y, or Z axis must be chosen depending on the problem and bonding orientation. In my case, at the moment all bondings are in the same direction and I just change the property in Primitives3D.py when I run the code.
Reproducing the code and changing the WhichAxis properties, different geometries are obtained.

Thank you very much in advance.

### Steps To Reproduce

import math, cmath

from pyaedt import Hfss
hfss= Hfss()
hfss.modeler.model_units ="mm"

## Global variables
mu0 	= 4*math.pi*1e-7
eps0 	= 8.85418*1e-12
c0 		= 1/math.sqrt(eps0*mu0)
mm 		= 1e-3
cm 		= 1e-2
GHz 	= 1e9

## Project variables
freq_res 	= 10 * GHz		# Hz
lambda_res 	= c0 / freq_res

## Variables
Ni = 3
No = 3
hfss["$msNfin"] = str(Ni)
hfss["$msNfout"] = str(No)
hfss["$msLin"] = "3.0mm"
hfss["$msWin"] = "1.62mm"
hfss["$msWf"] = "0.170mm"    
hfss["$msSf"] = "0.120mm"    
hfss["$msSin"] = "0.120mm"   
hfss["$msLf"] = "9.5mm"    
hfss["$msLinA"] = "1.0mm"    
hfss["$msT"] = "0.032mm"    

hfss['$msWinA'] = "($msNfin + $msNfout) * $msWf + ($msNfin + $msNfout -1) * $msSf"

hfss["$substrateThickness"] = "30mils"   
hfss["$substrateWidth"]     = "$msWinA * 7"  
hfss["$substrateLength"]    = "2 * $msLin + 2 * $msLinA + $msLf + $msSin" 
         
hfss["$bondDiameter"] = "0.025mm"
hfss["$bondHeight"] = "0.1mm"
hfss["$bondSep"] = "0.15mm"

## MS dimensions
#substrateThickness 	= Ex("30mils")	# mils
#substrateWidth 		= Ex("20mm")			# mm
#substrateLength 	= Ex("30mm")			# mm

substrateMaterial = "Rogers RO4350 (tm)"
metalMaterial     = "Copper"

projectName = "Test Project"
designName  = "Test Design"

## Create fingers
for i in range(0,Ni):
    fi=hfss.modeler.create_box(["-$msWinA/2 + 2 *" +  str(i) +" * ($msWf + $msSf)","$substrateThickness","$msLin + $msLinA"],
                                    ["$msWf","$msT", "$msLf"], 
                                     name="fingerIn", matname="pec")
    fi.color = (0,0,1)
    fi.transparency = 0.5
for i in range(0,No):
    fo=hfss.modeler.create_box(["-$msWinA/2 + (2 *" + str(i) +"+1) * ($msWf + $msSf)","$substrateThickness","$msLin + $msLinA + $msSin"],
                                    ["$msWf","$msT", "$msLf"], 
                                     name="fingerOut", matname="pec")
    fo.color = (0,0,1)
    fo.transparency = 0.5
    
# Create Wire bondings
for i in range(0,Ni-1):
    origin = ["-$msWinA/2 + $msWf/2 +"+ str(2*i) + " * ($msWf + $msSf)", 
              "$substrateThickness + $msT", 
              "$msLin + $msLinA +$msSin + $msLf - $bondSep"]
    endpos = ["-$msWinA/2 + $msWf/2 +"+ str(2*i) + " *($msWf + $msSf) + 2 * ($msSf + $msWf)", 
              "$substrateThickness + $msT", 
              "$msLin + $msLinA +$msSin + $msLf - $bondSep"]
    bi = hfss.modeler.create_bondwire(origin, endpos,
                                   	h1="$bondHeight", h2="0mm", 
                                    alpha=70, beta = 50, 
                                   	bond_type=2, diameter = "$bondDiameter", facets = 6,
                                    name="Bondi", matname="pec")
    
for i in range(0,No-1):
    origin = ["-$msWinA/2 + $msWf/2 + "+ str(2*i+1) + " * ($msWf + $msSf)", 
              "$substrateThickness + $msT", 
              "$msLin + $msLinA + $msSin + $bondSep"]
    endpos = ["-$msWinA/2 + $msWf/2 + "+ str(2*i+1) + " * ($msWf + $msSf) + 2 * ($msSf + $msWf)", 
              "$substrateThickness + $msT", 
              "$msLin + $msLinA +$msSin  + $bondSep"]
    bi = hfss.modeler.create_bondwire(origin, endpos,
                                   	h1 = "$bondHeight", h2="0mm", 
                                    alpha=70, beta = 50,   
                                   	bond_type = 2, diameter = "$bondDiameter", facets = 6,
                                    name = "Bondo", matname="pec")

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

PS C:\Users\erago\OneDrive - Universidad de Málaga\Diseño\HFSS_scripts\MCTxLine> python -m pip freeze
absl-py==1.1.0
appdirs==1.4.4
asttokens==2.2.1
astunparse==1.6.3
backcall==0.2.0
cachetools==5.2.0
certifi==2022.6.15
cffi==1.15.1
charset-normalizer==2.0.12
clr-loader==0.2.5
colorama==0.4.6
comm==0.1.2
cycler==0.11.0
debugpy==1.6.5
decorator==5.1.1
entrypoints==0.4
executing==1.2.0
flatbuffers==1.12
fonttools==4.33.3
fpdf==1.7.2
gast==0.4.0
gmsh==4.11.1
google-auth==2.8.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.47.0
h5py==3.7.0
hycohanz @ https://github.com/Pablo097/hycohanz/archive/devel.zip
idna==3.3
imageio==2.24.0
ipykernel==6.20.1
ipython==8.8.0
jedi==0.18.2
jupyter_client==7.4.8
jupyter_core==5.1.3
keras==2.9.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.3
libclang==14.0.1
Markdown==3.3.7
matplotlib==3.5.2
matplotlib-inline==0.1.6
nest-asyncio==1.5.6
numpy==1.23.0
oauthlib==3.2.0
opt-einsum==3.3.0
packaging==21.3
pandas==1.4.3
parso==0.8.3
pickleshare==0.7.5
Pillow==9.1.1
platformdirs==2.6.2
plumbum==1.8.1
pooch==1.6.0
progress==1.6
progress-bar==8
progressbar==2.5
progressbar2==4.0.0
prompt-toolkit==3.0.36
protobuf==3.19.4
psutil==5.9.4
pure-eval==0.2.2
pyaedt==0.6.82
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
Pygments==2.14.0
pyparsing==3.0.9
python-dateutil==2.8.2
python-utils==3.3.3
pythonnet==3.0.1
pytz==2022.1
pyvista==0.37.0
pywin32==305
pyzmq==24.0.1
quantiphy==2.19
requests==2.28.0
requests-oauthlib==1.3.1
rpyc==5.3.1
rsa==4.8
scikit-rf==0.24.1
scipy==1.10.0
scooby==0.7.0
six==1.16.0
stack-data==0.6.2
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.9.1
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
termcolor==1.1.0
tornado==6.2
traitlets==5.8.1
typing_extensions==4.2.0
urllib3==1.26.9
vtk==9.2.5
wcwidth==0.2.5
Werkzeug==2.1.2
wrapt==1.14.1

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Close #3214

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Comment by enriquemarquez** [comment]
Dear Samuel,
thank you very much for your work. I suppose it is just a problem of mine, but when you say add axis, what do you mean? I try to read the package but I can't find how to proceed. Sorry for any inconvenience.
Enrique.

**Comment by Samuelopez-ansys** [comment]
@enriquemarquez I have added the option in the method to pass the axis ("X", "Y", "Z"), as you suggested in the issue. The key of the argument is 'cs_axis' to be aligned with other PyAEDT methods.

Please be aware, if your bondwire definition (start definition, stop) is not oriented in the defined axis, the generated bondwire will be not correct in the 3DModeler. If the axis orientation is tricky, I would define the bondwire in the Z axis, and the I would rotate it with a coordinate system, this is more robust.

**Comment by enriquemarquez** [comment]
Thank you very much @Samuelopez-ansys . I will proceed with rotation. I think it's simpler. 
BTW, as I am working in free space material characterization, I read your report from Yebes. Additionally, I have been using hycohanz from Pablo who is a Ph.D. student in our research group. He said to me he met you last year. ¡Qué pequeño es el mundo!

**Comment by Samuelopez-ansys** [comment]
@enriquemarquez Indeed small world :)

Pablo has my email, please reach out if you want to discuss your current projects and how your department could contribute to PyAEDT.
